portfolio/src/main/webapp/index.html

```diff
@@ -62,6 +64,15 @@ <h2>Your comments</h2>
         <ol id="Comments">
         </ol>
       <button onclick="deleteUserComments()">Delete your comments</button>
+      <button>Translate your comment!</button>
+
+      <hr>
+
+      <h2>Maps!</h2>
+        <div id="map"></div>
+        <p>Look at the world!</p>
```

Comment:
Indented too much

Reply:
I believe the indentation here is 2 spaces